### PR TITLE
Patrick steveson

### PR DESCRIPTION
**Pull request by Mrsteveson**
Components Built.
Minor Styling Layout done.
updateList Function works.

working on the line-through.
**then** 
clear complete functionality.

### CODE REVIEW

**Comment by Mrsteveson** [comment]
As of the latest commit, i can toggle  completed: true or false, and clearCompleted will filter properly. still working on getting the line-through to apply

File: src/App.js [hunk 1]
```diff
@@ -1,16 +1,141 @@
...
+  //     }
+  //   });
+
+  //   this.setState({
+  //     todoList: newTodoList
+  //   });
+  // }
+
+  // both version of finished work.
+  // trying to simplify it so it makes more sense to me.
```
Comment (NickStrick) [inline src/App.js:82]: awesome! always find new ways to do the same thing, a creative mindset will work wonders for you.

File: src/components/TodoComponents/Todo.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import './Todo.css';
+
+const Todo = props => {
+    return(
+        <div>
+            <p 
+            className = {props.todoList.completed ? 'finished' : 'todoContent'}
```
Comment (NickStrick) [inline src/components/TodoComponents/Todo.js:8]: love seeing this synaxx! use it more! haha

**Review by NickStrick — APPROVED** [review]
Looks amazing! really great stretch too! and css! all around amazing, your very well prepared for the sprint this week! keep that knowledge train going and never stop improving,  theres always something new to learn!

**Pull request merged**